lib/srv6.h

```diff
@@ -174,18 +174,24 @@ struct srv6_locator_chunk {
  * https://www.iana.org/assignments/segment-routing/segment-routing.xhtml
  */
 enum srv6_endpoint_behavior_codepoint {
-	SRV6_ENDPOINT_BEHAVIOR_RESERVED         = 0x0000,
-	SRV6_ENDPOINT_BEHAVIOR_END              = 0x0001,
-	SRV6_ENDPOINT_BEHAVIOR_END_X            = 0x0005,
-	SRV6_ENDPOINT_BEHAVIOR_END_DT6          = 0x0012,
-	SRV6_ENDPOINT_BEHAVIOR_END_DT4          = 0x0013,
-	SRV6_ENDPOINT_BEHAVIOR_END_DT46         = 0x0014,
-	SRV6_ENDPOINT_BEHAVIOR_END_NEXT_CSID    = 0x002B,
-	SRV6_ENDPOINT_BEHAVIOR_END_X_NEXT_CSID  = 0x002C,
-	SRV6_ENDPOINT_BEHAVIOR_END_DT6_USID     = 0x003E,
-	SRV6_ENDPOINT_BEHAVIOR_END_DT4_USID     = 0x003F,
-	SRV6_ENDPOINT_BEHAVIOR_END_DT46_USID    = 0x0040,
-	SRV6_ENDPOINT_BEHAVIOR_OPAQUE           = 0xFFFF,
+	SRV6_ENDPOINT_BEHAVIOR_RESERVED = 0x0000,
```

Comment:
It'd be better not to un-do the existing clear formatting?

Reply:
well, git clang will complain, no?

Reply:
clang is wrong about several things - like this.
the goal is clear, readable code - if clang is wrong, then ... it's wrong.